### DISCUSSION TITLE
PWA not working with nginx

### DISCUSSION THREAD

**Discussion by Jwerfel** [coder/code-server#2075] [Q&A] [answered]
I have code-server installed on a Ubuntu VM running behind nginx with SSL offloading.  Basic functionality is working fine however when I run in Chrome I don't get an option to run as a PWA.  My code-server instance runs on my domain in the /code path

I looked in the console and it shows:

`The path of the provided scope ('/code') is not under the max scope allowed (set by Service-Worker-Allowed: '/code/'). Adjust the scope, move the Service Worker script, or use the Service-Worker-Allowed HTTP header to allow the scope.`

`Uncaught (in promise) DOMException: Failed to register a ServiceWorker for scope ('https://mydomain.com/code') with script ('https://mydomain.com/code/static/de41646fc402b968ca6d555fdf2da7de9554d28a/usr/lib/code-server/dist/serviceWorker.js'): The path of the provided scope ('/code') is not under the max scope allowed (set by Service-Worker-Allowed: '/code/'). Adjust the scope, move the Service Worker script, or use the Service-Worker-Allowed HTTP header to allow the scope.`

So is there a way to move the service worker or adjust a setting or something to make this work?

code-server: v3.5.0

Thank you

**Comment by code-asher** [comment, marked as answer]
Thanks for the report! I opened an issue: https://github.com/cdr/code-server/issues/2076

Will fix asap so it can get in the next release.

**Comment by Jwerfel** [comment]
Thanks for the response. I applied the change you made to my local system and confirmed it works now!

**Reply by code-asher** [reply]
Thanks for testing it!
